src/BreakfastSerializer/Serializer.php

```diff
@@ -153,4 +159,24 @@ protected function cleanVariableName($variableName, $containingClass)
         return trim($cleanedName);
     }
 
+    /**
+     * @param string $propertyName
+     * @param string $className
+     * @return bool
+     */
+    public function isExcluded($propertyName, $className)
+    {
+        if (true === isset($this->getConfiguration()['exclusions'][$className]['excludeVariables'])) {
```

Comment:
In your `composer.json` file you have `"php": ">=5.3.2"`, is this not `5.4` and above?